app/facades/tourist_sights_facade.rb

```diff
@@ -0,0 +1,12 @@
+class TouristSightsFacade
+
+  def self.find_sights(country)
+    country = GeocodingService.capital_info(capital, country)
```

Comment:
This is not working because you are calling to `capital` but have never defined that variable.

In order to get sights for a country's capital, we need to: 

1. find the lat and long for the capital of that country
2. pass off the lat and long to the Places API to find tourist sights around the capital

In order to do #1 above, we can utilize code that we already have available to us. The CountryFacade has a `capital_info` method that returns a Capital object. That Capital object has the lat and long for the capital.